x-pack/legacy/plugins/lens/public/indexpattern_datasource/indexpattern.tsx

```diff
@@ -198,15 +207,73 @@ export function getIndexPatternDatasource({
       );
     },
 
-    getPublicAPI({
-      state,
-      setState,
-      layerId,
-      dateRange,
-    }: PublicAPIProps<IndexPatternPrivateState>) {
+    renderDimensionTrigger: (
```

Comment:
For reviewers: `renderDimensionTrigger` is only rendering the clickable link. It has almost no behavior any more.